google/cloud/aiplatform/featurestore/featurestore.py

```diff
@@ -301,24 +302,253 @@ def list_entity_types(
+                name.
+
+                This value may be up to 60 characters, and valid characters
+                are ``[a-z0-9_]``. The first character cannot be a number.
+
+                The value must be unique within the project and location.
+            online_store_fixed_node_count (int):
+                Required. Config for online serving resources.
+            labels (Dict[str, str]):
+                Optional. The labels with user-defined
+                metadata to organize your Featurestore.
+                Label keys and values can be no longer than 64
+                characters (Unicode codepoints), can only
+                contain lowercase letters, numeric characters,
+                underscores and dashes. International characters
+                are allowed.
+                See https://goo.gl/xmQnxf for more information
+                on and examples of labels. No more than 64 user
+                labels can be associated with one
+                Featurestore(System labels are excluded)."
+                System reserved label keys are prefixed with
+                "aiplatform.googleapis.com/" and are immutable.
+            project (str):
+                Optional. Project to create EntityType in. If not set, project
+                set in aiplatform.init will be used.
+            location (str):
+                Optional. Location to create EntityType in. If not set, location
+                set in aiplatform.init will be used.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to create EntityTypes. Overrides
+                credentials set in aiplatform.init.
+            request_metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as metadata.
+            request_metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as metadata.
+            encryption_spec (str):
+                Optional. Customer-managed encryption key
+                spec for data storage. If set, both of the
+                online and offline data storage will be secured
+                by this key.
+            sync (bool):
+                Optional. Whether to execute this creation synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+
+        Returns:
+            Featurestore - Featurestore resource object
+
+        """
+        if labels:
+            utils.validate_labels(labels)
+
+        gapic_featurestore = gca_featurestore.Featurestore(
+            labels=labels,
+            online_serving_config=gca_featurestore.Featurestore.OnlineServingConfig(
+                fixed_node_count=online_store_fixed_node_count
```

Comment:
Just a head up, we are planing to have auto-scaling to public preview. Which mean, setting this count will need to change to setting scaling_config (which may need 2 numbers). We can revisit this when it's done, just fyi for now.

Reply:
Noted